app/controllers/agents/relatives_controller.rb

```diff
@@ -48,6 +56,10 @@ def destroy
 
   private
 
+  def from_modal?
```

Comment:
let's move this to AgentAuthController or ApplicationController as it's used in multiple controllers and is pretty generic